akka-edge-docs/src/main/paradox/feature-summary.md

```diff
@@ -153,6 +153,6 @@ be used for the replicas. For example, Postgres in the Cloud and H2 at the edge.
 
 ### Learn more
 
-* FIXME link to guide
+* @ref[Hands-on guide for implementing an Akka Edge application](guide.md) (Drone Charging Station)
```

Comment:
I guess we don't want them to jump directly to step 5

Reply:
I think top guide page is good